javascript/react/good-to-know/immutability-helpers-in-react.md

```diff
@@ -25,52 +25,73 @@ aspects:
 
 ---
 
-# Immutability helpers in **React**
+# Immutability helpers in React
 
 ---
 ## Content
 
-Opting out for an immutable style of data management, a significant speed up can be achieved by easily implementing a fast `shouldComponentUpdate()` lifecycle method.
+When opting in for an immutable style of data management, a significant speedup can be achieved by easily implementing a fast `shouldComponentUpdate()` lifecycle method. To do this, you would need a method that aids with comparing *versions* of your component's `state`.
```

Comment:
I think we should remove "easily" and "fast" from the first sentence. First one is subjective and second one unnecessary. 

> When opting in for an immutable style of data management, a significant speedup can be achieved by implementing a `shouldComponentUpdate()` lifecycle method.